### DISCUSSION TITLE
How to fully "rebind" a button?

### DISCUSSION THREAD

**Discussion by Jacob-077** [AntiMicroX/antimicrox#397] [Q&A]
Hello, 

I want to make the "Dpad - Up" button on my controller to act ONLY as the Spacebar key.

When I normally try to do so, both keys co-exist instead of the new key's action replacing the old one, so now when I press "Up" on the Dpad, it does 2 actions.

What I want is for the Spacebar key to fully replace the Up button so that it doesn't produce conflicts.

Thank.

**Comment by pktiuk** [comment]
Are you talking about some way of intercepting gamepad button presses?

As far as I understood when you press dpad-UP button (mapped to space) game receives two presses (space press and dpad press at once)

EDIT:  
In case of Linux check: https://github.com/AntiMicroX/antimicrox/wiki/Methods-for-Disabling-Joystick-Reading-in-Linux

**Reply by pktiuk** [reply]
@Jacob-Darka   
Do I understand correctly?

**Reply by Jacob-077** [reply]
Yes, I want the game to receive only one press which in this case is the space key and totally ignore what the dpad button actually does in-game.

Or is that not possible?

**Reply by pktiuk** [reply]
What is your system?   
If Linux then you can use instructions from wiki: https://github.com/AntiMicroX/antimicrox/wiki/Methods-for-Disabling-Joystick-Reading-in-Linux

**Reply by Jacob-077** [reply]
I'm on Windows.
